inc/languages/english/global.lang.php

```diff
@@ -441,13 +441,14 @@
 
 $l['invalid_post_code'] = "Authorization code mismatch. Are you accessing this function correctly? Please go back and try again.";
 $l['invalid_nocaptcha'] = "Please solve the reCAPTCHA to verify that you're not a robot.";
+$l['invalid_hcaptcha'] = "Please solve the hCaptcha to verify that you're not a robot.";
 $l['invalid_captcha_verify'] = "The image verification code that you entered was incorrect. Please enter the code exactly how it appears in the image.";
 $l['image_verification'] = "Image Verification";
 $l['human_verification'] = "Human Verification";
 $l['verification_note'] = "Please enter the text contained within the image into the text box below it. This process is used to prevent automated spam bots.";
-$l['verification_note_nocaptcha'] = "Please tick the checkbox that you see below. This process is used to prevent automated spam bots.";
+$l['verification_note_nocaptcha'] = $l['verification_note_hcaptcha'] = "Please tick the checkbox that you see below. This process is used to prevent automated spam bots.";
 $l['verification_subnote'] = "(case insensitive)";
-$l['invalid_nocaptcha_transmit'] = "An error occurred with the human verification. Please try again.";
+$l['invalid_nocaptcha_transmit'] = $l['invalid_hcaptcha_transmit'] = "An error occurred with the human verification. Please try again.";
```

Comment:
If they use the same string then why bother using two different language strings? Why not simply use `invalid_nocaptcha_transmit`?

Reply:
That's definitely a language system design short sight. Anything could be different in usage and could be more descriptive, though some of the entries are having the same content.

I would try following:
```
$l['invalid_nocaptcha_transmit'] = "An error occurred with the human verification by noCaptcha. Please try again.";
$l['invalid_hcaptcha_transmit'] = "An error occurred with the human verification by hCaptcha. Please try again.";
```

I think I'll start a discussion about the re-design for the language system, especially on how the entry is named.

Reply:
> If they use the same string then why bother using two different language strings? Why not simply use `invalid_nocaptcha_transmit`?

As this is a String for reCAPTCHA, I tried avoiding it in the usage of hCaptcha.

> I would try following:
> 
> ```
> $l['invalid_nocaptcha_transmit'] = "An error occurred with the human verification by noCaptcha. Please try again.";
> $l['invalid_hcaptcha_transmit'] = "An error occurred with the human verification by hCaptcha. Please try again.";
> ```
Sounds good to me 👍